content/docs/resiliency/release/_index.md

```diff
@@ -8,13 +8,16 @@ Description: >
 
 ## Release Notes - CSM Resiliency 1.2.0
 
-- Added support for PowerScale CSI Driver.
+- Support for CSI Driver for PowerScale.
```

Comment:
Can you update his table too : https://github.com/dell/csm-docs/blob/main/content/docs/_index.md#csm-modules-support-matrix-for-dell-csi-drivers ?
thx

Reply:
@coulof This is already addressed. Please look at release-1.3 branch.
https://github.com/dell/csm-docs/blob/release-1.3/content/docs/_index.md